ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ConfigManagementServiceIT.scala

```diff
@@ -90,14 +80,6 @@ final class ConfigManagementServiceIT extends LedgerTestSuite {
         response3.timeModel.equals(response1.timeModel),
         "Restoring the original time model failed",
```

Comment:
Can you update this message as well? The intention probably changed but the explanation is wrong